spark/sql-13/build.gradle

```diff
@@ -55,10 +55,10 @@ configurations.all { Configuration conf ->
 tasks.withType(ScalaCompile) { ScalaCompile task ->
     task.sourceCompatibility = project.ext.minimumRuntimeVersion
     task.targetCompatibility = project.ext.minimumRuntimeVersion
+    task.options.forkOptions.executable = new File(project.ext.runtimeJavaHome, 'bin/java').canonicalPath
```

Comment:
Is there a compelling reason to use `canonicalPath` here? There's a performance cost to doing so as resolving a canonical path involves file IO at configuration time.

Reply:
I'll put up a quick fix for this then. I was following the same approach that the Gradle internals are doing, but I suppose if they're already getting the canonical file, we don't have to here.